releng/target-platform/target-platform.target

```diff
@@ -74,6 +73,10 @@
             <repository location="https://download.eclipse.org/jdtls/milestones/1.5.0/repository/"/>
             <unit id="org.eclipse.jdt.ls.core" version="1.5.0.202110191539"/>
         </location>
+        <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
```

Comment:
Confused what this has to do with this PR?  Didn't we already make a similar change?  Might you need to rebase?

Reply:
yes, needs a re-base to pick this up officially - had made it a local change